setup.cfg

```diff
@@ -0,0 +1,19 @@
+[metadata]
+name = skt
+description = "Sonic Kernel Testing"
+long_description = file: README.md
+version = 1
+author = Red Hat, Inc.
+license = GPLv2+
+
+[options]
+packages = find:
+include_package_data = True
```

Comment:
Could you please add comments for each of these options, explaining what they do? I failed to find documentation for these quickly, and comments might be useful for others too. Unless this is obvious for all other Python developers, of course. In that case a pointer to the documentation here would suffice :) Thanks!

Reply:
Yes, I'll do that now.